Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift

```diff
@@ -1,85 +1,82 @@
 //
-//  ViewController.swift
-//  ThumbnailExample
+//  BannerManager.swift
+//  Direct Integration Sample
 //
-//  Copyright © 2020 Ogury Co. All rights reserved.
+//  Created by Fernand Peng on 09/11/2021.
```

Comment:
remark (blocking) : remove all 
`Created by Fernand Peng on 09/11/2021.`
on the top of file, you can write : 
` Copyright © 2020 Ogury Co. All rights reserved.`

Reply:
Updated in 750d11cae96c04f3272b4a34935c4ea6ed56a689